testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/servlet/OfflineServletsAdapterTest.java

```diff
@@ -1,34 +1,30 @@
 package org.keycloak.testsuite.adapter.servlet;
 
 import javax.ws.rs.core.UriBuilder;
-import java.util.List;
+import java.util.*;
```

Comment:
Please don't use star imports.

Reply:
Thanks @vmuzikar I removed the asterisk imports and fixed the flaky test the PR previously introduced. Could you take a look now please ?